Content.Client/MainMenu/MainMenu.cs

```diff
@@ -26,6 +26,8 @@ public sealed class MainScreen : Robust.Client.State.State
         [Dependency] private readonly IResourceCache _resourceCache = default!;
         [Dependency] private readonly IUserInterfaceManager _userInterfaceManager = default!;
 
+        private readonly ISawmill _sawmill = Logger.GetSawmill("mainmenu");
+
```

Comment:
Is this already a dependency? I don't see ILogManager anywhere.

Reply:
Uh, it's Logger class.

https://github.com/space-wizards/RobustToolbox/blob/fc1cca4f48f2f2d3fbf41aa42b80b4e43b1095a4/Robust.Shared/Log/Logger.cs#L16

Reply:
Can't tell on github if it's the static or not.

Reply:
> Can't tell on github if it's the static or not.

https://github.com/space-wizards/RobustToolbox/blob/4920ecaa64310a1680c388dd21c7ee7a43590bc7/Robust.Shared/Log/Logger.cs#L16